geopandas/base.py

```diff
@@ -97,25 +90,14 @@ def _delegate_geo_method(op, this, *args, **kwargs):
 class GeoPandasBase(object):
     _sindex = None
     _sindex_generated = False
+    sindex_cls = None
```

Comment:
I think this is a left-over from the keyword you removed for now?

Reply:
Yes, great catch, my mistake.